mobile/src/pages/Register.tsx

```diff
@@ -11,6 +11,27 @@ export default function Register({ navigation }) {
   const [password, setPassword] = useState("");
   const [confirmPassword, setConfirmPassword] = useState("");
   const [isPending, setIsPending] = useState(false);
+  const [error, setError] = useState("");
+
+  //ok but shouldn't this be a hook? anyways it works regardless
+  let isDisabled = false;
```

Comment:
Yes, this might work but will definitely have unreliable behavior at some point. Please use `useState`.

Reply:
react yelled at me for re-renders when I made this a hook, does that mean that theres a better way to just write the logic that isdisabled is used in?

Reply:
Yeah just remove all of this stuff